crates/router/src/core/payment_methods/vault.rs

```diff
@@ -506,10 +506,15 @@ pub async fn create_tokenize(
     )
     .change_context(errors::ApiErrorResponse::InternalServerError)?;
 
-    let (public_key, private_key) = get_locker_jwe_keys(&state.kms_secrets)
-        .await
-        .change_context(errors::ApiErrorResponse::InternalServerError)
-        .attach_printable("Error getting Encryption key")?;
+    let (public_key, private_key) = get_locker_jwe_keys(
```

Comment:
why are we making this change is it needed?

Reply:
here again, as kms is made optional, we need to pull it out of the option...hence the ok_or